audformat/core/utils.py

```diff
@@ -1003,6 +1003,27 @@ def map_file_path(
 ) -> pd.Index:
     r"""Apply callable to file path in index.
 
+    Relies on :meth:`pandas.Index.map`,
+    which can be slow.
+    If speed is crucial,
+    consider to change the index directly.
+    In the following example we prefix every file with a folder
+    and add a new extension:
```

Comment:
I'm now fine with staying with the example, but maybe we should add a link to the corresponding utility function that can achieve the same?

```suggestion
    and add a new extension,
    compare also :func:`audformat.utils.expand_file_path`
    and :func:`audformat.utils.replace_file_extension`:
```